meta-intel-edison-bsp/recipes-bsp/u-boot/files/edison.env

```diff
@@ -16,7 +16,7 @@ ota_update_max_retries=3
 audio_codec_name=audio_codec="dummy"
 do_audio_support=setenv audio_support platform_mrfld_audio.${audio_codec_name}
 do_compute_target=if itest.b ${first_install_retry} -gt ${first_install_max_retries} || itest.b ${ota_update_retry} -gt ${ota_update_max_retries}; then echo "Switch to Rescue target"; setenv bootargs_target rescue; saveenv; fi
-mmc-bootargs=run do_bootargs_rootfs; run do_audio_support; setenv bootargs ${bootargs_common} ${bootargs_rootfs} ${bootargs_console} ${bootargs_debug} systemd.unit=${bootargs_target}.target hardware_id=${hardware_id}
```

Comment:
This env for U-Boot is supposed to provide hardwire_id to kernel command line. But afaikt always hardware_id=00. I am assuming that is wrong anyway and just dropped it. Correct?

Reply:
Have you checked if any tool is actually using it for something more serious than debug printing?

Reply:
I see it is used in U-Boot: `setenv dfu_alt_info "ifwi${hardware_id} raw 0 8192 mmcpart 1;ifwib${hardware_id} raw 0 8192 mmcpart 2"`, but no idea why it was passed on the kernel command line. Something SFI related?

Reply:
No idea, that's why I asked to check the user space tools in the original image to see if any of them is using it (it needs to be checked in source code, because some C code might parse it).